Dockerfile.testing

```diff
@@ -1,24 +1,21 @@
 # For building gcr.io/forgerock-io/secret-agent-testing:latest
 
-FROM openjdk:11-jdk-slim
+FROM openjdk:11-jre-slim
 ENV CGO_ENABLED=0 GOOS=linux GOARCH=amd64
-CMD ["bash"]
-RUN apt-get update && apt-get install -y curl git-core
-RUN curl -O https://dl.google.com/go/go1.13.10.linux-amd64.tar.gz && \
-    SUM=$(sha256sum go1.13.10.linux-amd64.tar.gz | awk '{print $1}') && \
+RUN apt-get update && apt-get install -y curl git-core make && apt-get clean all
+RUN curl -LO https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz && \
+    SUM=$(sha256sum go1.14.2.linux-amd64.tar.gz | awk '{print $1}') && \
     if [ "$SUM" != "8a4cbc9f2b95d114c38f6cbe94a45372d48c604b707db2057c787398dfbf8e7f"]; then echo "Failed checksum"; exit 1; fi && \
```

Comment:
Seems like the checksum should've changed with a different version of Go? If this succeeded, then something's wrong with our checksum validation logic.

Reply:
I will take a look at the checksum validation and update the PR. There's definitely a problem.
Good catch! 👍

Reply:
fixed. just a small bug in the if statement